back/engines/free/frontend/app/services/frontend/url_service.rb

```diff
@@ -89,6 +89,10 @@ def reset_password_url(token, options = {})
       "#{home_url(options)}/reset-password?token=#{token}"
     end
 
+    def reset_confirmation_code_url(options = {})
```

Comment:
Maybe we should also consider moving this into the engine? But not a big deal in my opinion. Actually, I'm starting to wonder if it's the right choice to put user confirmation in a separate engine as this adds some complications and is strongly entangled with the core application code.